components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -1234,11 +1234,13 @@ def enrolled_with(enrollment)
         evidence = self.send(evidence_type)
         next unless evidence.present?
         enrollment_member = enrollment.hbx_enrollment_members.where(:applicant_id => family_member_id).first
-        aptc_or_csr_used = enrollment_member.applied_aptc_amount > 0 || ['csr_73', 'csr_87', 'csr_94', 'csr_limited'].include?(csr_eligibility_kind)
+        aptc_or_csr_used = enrollment_member.applied_aptc_amount > 0 || ['03', '04', '05', '06'].include?(enrollment.product.csr_variant_id)
```

Comment:
@j1joey can you use this here? [EligibilityDetermination::CSR_KIND_TO_PLAN_VARIANT_MAP.default].exclude?(enrollment.product.csr_variant_id)

Reply:
I dont think we can because that mapping includes the csr_0 mapping, which we want to exclude